stats.lua

```diff
@@ -472,6 +530,7 @@ local function add_video(s)
 
     append_property(s, "video-params/gamma", {prefix="Gamma:", suffix=hdrinfo})
     append_property(s, "packet-video-bitrate", {prefix="Bitrate:", suffix=" kbps"})
+    append_filters(s, "vf", "Video Filters:")
```

Comment:
Better use only "Filters:". It's within the "Video" category so it should be obvious, also makes the label shorter.